P/Pythia8/build_tarballs.jl

```diff
@@ -0,0 +1,42 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Pythia8"
+version = v"8.244.0"
+
+# Collection of sources required to complete build
+sources = [
+    "http://home.thep.lu.se/~torbjorn/pythia8/pythia8244.tgz" =>
+    "e34880f999daf19cdd893a187123927ba77d1bf851e30f6ea9ec89591f4c92ca",
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd pythia8244/
+./configure --prefix=${prefix} --enable-shared --enable-64bit
+make -j${nproc}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Linux(:x86_64, libc=:glibc),
+    Linux(:powerpc64le, libc=:glibc),
+    MacOS(:x86_64)
+]
+
```

Comment:
https://dev.azure.com/JuliaPackaging/Yggdrasil/_build/results?buildId=1081&view=logs&j=2b277407-6ab0-5e15-108f-8beae357ef17&t=551d77a4-d348-5c2d-a045-3b696b2c9100&l=1803
```
[ Info: /workspace/P/Pythia8/build/powerpc64le-linux-gnu/bbE9s0Ue/destdir/lib/libpythia8.so locks us to cxx11
┌ Warning: /workspace/P/Pythia8/build/powerpc64le-linux-gnu/bbE9s0Ue/destdir/lib/libpythia8.so contains std::string values!  This causes incompatibilities across the GCC 4/5 version boundary.  To remedy this, you must build a tarball for both GCC 4 and GCC 5.  To do this, immediately after your `platforms` definition in your `build_tarballs.jl` file, add the line:
│ 
│     platforms = expand_cxxstring_abis(platforms)
└ @ BinaryBuilder ~/.julia/dev/BinaryBuilder/src/auditor/compiler_abi.jl:189
```
```suggestion
platforms = expand_cxxstring_abis(platforms)
```